src/components/auth/Register.jsx

```diff
@@ -51,145 +51,149 @@ const Register = () => {
-                id="password1"
-                placeholder={t('register.passwordPlaceholder')}
-                onChange={handleChange('password1')}
-                value={values.password1}
-                disabled={isLoading}
-                error={errors.password1 && touched.password1}
-                InputProps={{
-                  startAdornment: (
-                    <InputAdornment position="start">
-                      <Tooltip title={t('register.passwordPrompt')}>
-                        <VpnKeyIcon />
-                      </Tooltip>
-                    </InputAdornment>
-                  ),
-                }}
-                helperText={errors.password1 && touched.password1 && errors.password1}
-              />
-
-              <TextField
-                className="content-container__text-field"
-                type="password"
-                id="password2"
-                placeholder={t('register.passwordConfirmationPlaceholder')}
-                onChange={handleChange('password2')}
-                value={values.password2}
-                disabled={isLoading}
-                error={errors.password2 && touched.password2}
-                InputProps={{
-                  startAdornment: (
-                    <InputAdornment position="start">
-                      <Tooltip title={t('register.passwordPrompt')}>
-                        <VpnKeyIcon />
-                      </Tooltip>
-                    </InputAdornment>
-                  ),
-                }}
-                helperText={errors.password2 && touched.password2 && errors.password2}
-              />
-              <FormControlLabel
-                control={
-                  <Checkbox
-                    checked={checked}
-                    color="primary"
-                    onChange={handleChange}
-                    inputProps={{'aria-label': 'primary checkbox'}}
-                    onClick={() => {
-                      setChecked(!checked);
-                    }}
-                  />
-                }
-                label={
-                  <p className="register__regulations">
-                    {t('register.agreementText')}
-                    <a target="_blank" className="agreement-link" rel="noopener noreferrer" href={REACT_APP_REGULATIONS}>
-                      {t('register.agreementThisLink')}.
-                    </a>
-                  </p>
-                }
-              />
-
-              <Button
-                variant="contained"
-                disabled={!checked || isLoading}
-                onClick={handleSubmit}
-                className="register__button">
-                {t('register.button')}
-              </Button>
-            </div>
-          )}
-        </Formik>
-      </div>
-
-      <div className="register__bottom">
-        <AuthBottomPanel linkText={t('register.login')} link={paths.LOGIN} />
-      </div>
-    </AuthContainer>
+      <Loader isLoading={isLoading} />
```

Comment:
Ten formater za każdym razem robi porządek inaczej. Może to jest uzależnione od platformy?